pkg/syncer/spec/spec_process_test.go

```diff
@@ -695,7 +694,7 @@ func TestSyncerProcess(t *testing.T) {
 				),
 			},
 		},
-		"SpecSyncer with AdvancedScheduling, deletion: object exist downstream": {
```

Comment:
Can we gather the deletion test cases for the non-advanced use case so that they follow each other ?

Reply:
done